account-compression/sdk/src/merkle-tree/index.ts

```diff
@@ -143,11 +172,22 @@ export class MerkleTree {
       }
       if (verbose) console.log(`node ${i} ${new PublicKey(node).toString()}`);
     }
+    return node;
+  }
+
+  static verify(
+    root: string,
+    merkleTreeProof: MerkleTreeProof,
+    verbose = false
+  ): boolean {
+    const node = MerkleTree.hashProof(merkleTreeProof, verbose);
     const rehashed = new PublicKey(node).toString();
     const received = new PublicKey(root).toString();
     if (verbose) console.log(`hashed ${rehashed} got ${received}`);
```

Comment:
nit: can you move this log to after your error? otherwise you'll get almost the same log twice for incorrect proofs in verbose mode

Reply:
Fixed!